server/api/controllers/instances/instancesController.js

```diff
@@ -165,48 +168,56 @@ function putInstanceMaintenance(req, res, next) {
   let name = null;
 
   co(function* () {
-    let instance = yield Instance.getById(id);
-    const instanceIds = [id];
-    const accountName = instance.AccountName;
-    const autoScalingGroupName = instance.getAutoScalingGroupName();
-    const environmentName = instance.getTag('Environment');
-    name = environmentName;
 
-    /**
-     * Update ASG IPS table
-     */
-    let entry = yield asgips.get(accountName, { AsgName: 'MAINTENANCE_MODE' });
-    let ips = JSON.parse(entry.IPs);
-    if (enable === true) {
-      ips.push(instance.PrivateIpAddress);
-      ips = _.uniq(ips);
-    } else {
-      _.pull(ips, instance.PrivateIpAddress);
-    }
-    yield asgips.put(accountName, { AsgName: 'MAINTENANCE_MODE', IPs: JSON.stringify(ips) });
-
-    /**
-     * Put instance to standby on AWS
-     */
-    let handler = enable ? EnterAutoScalingGroupInstancesToStandby : ExitAutoScalingGroupInstancesFromStandby;
     try {
-      yield handler({ accountName, autoScalingGroupName, instanceIds });
-    } catch (err) {
-      if (err.message.indexOf('is not in Standby') !== -1 || err.message.indexOf('cannot be exited from standby as its LifecycleState is InService') !== -1) {
-        logger.warn(`ASG ${autoScalingGroupName} instance ${id} is already in desired state for ASG Standby: ${enable}`);
+      let instance = yield Instance.getById(id);
+      const instanceIds = [id];
+      const accountName = instance.AccountName;
+      const autoScalingGroupName = instance.getAutoScalingGroupName();
+      const environmentName = instance.getTag('Environment');
+      name = environmentName;
+
+      /**
+       * Update ASG IPS table
```

Comment:
I know you probably didn't do this, but it would be nice to remove the comment and move the code block to a method...